<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Statisticwindow</class>
 <widget class="QWidget" name="Statisticwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Statisticwindow{
	background: rgb(188,188,188);
}
QPushButton{
	outline : 0px;
	font: bold 14px;
	color: white;
	background: rgb(68, 68, 68);
	border-radius: 7px;
}

QPushButton:pressed{ 
	background:rgb(134, 134, 134) ;
}
QPushButton:disabled {
	background:rgb(134, 134, 134) ;	
	color: rgb(68, 68, 68);
}</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>894</x>
     <y>510</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="levelProggresBar1">
   <property name="geometry">
    <rect>
     <x>178</x>
     <y>70</y>
     <width>141</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar
{
border: 2px solid black;
border-bottom-right-radius: 9px;
border-bottom-left-radius: 9px;
background-color:rgb(131, 131, 131);
color: white;
font: bold 16px;
padding: 4px
}
QProgressBar::chunk 
{
background-color: rgb(44, 135, 255);
height: 3px;
border-bottom-right-radius: 3px;
border-bottom-left-radius: 3px;
} 

</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QProgressBar" name="levelProggresBar2">
   <property name="geometry">
    <rect>
     <x>358</x>
     <y>70</y>
     <width>141</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar
{
border: 2px solid black;
border-bottom-right-radius: 9px;
border-bottom-left-radius: 9px;
background-color:rgb(131, 131, 131);
color: white;
font: bold 16px;
padding: 4px
}
QProgressBar::chunk 
{
background-color: rgb(44, 135, 255);
height: 1px;
} 

</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textVisible">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>823</x>
     <y>205</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 14px;
color: white;</string>
   </property>
   <property name="text">
    <string>80</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>764</x>
     <y>92</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 14px;
color: white;</string>
   </property>
   <property name="text">
    <string>60</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>626</x>
     <y>92</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 14px;
color: white;</string>
   </property>
   <property name="text">
    <string>40</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>693</x>
     <y>79</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 14px;
color: white;</string>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>635</x>
     <y>318</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 14px;
color: white;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>578</x>
     <y>142</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 14px;
color: white;</string>
   </property>
   <property name="text">
    <string>30</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>752</x>
     <y>319</y>
     <width>31</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 14px;
color: white;</string>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLabel" name="temperatureValue">
   <property name="geometry">
    <rect>
     <x>656</x>
     <y>192</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: black;</string>
   </property>
   <property name="text">
    <string>99</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>579</x>
     <y>269</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 14px;
color: white;</string>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QDial" name="temperatureBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>586</x>
     <y>102</y>
     <width>231</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QDial{
background-color:rgb(45, 45, 45);
}</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="sliderPosition">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="notchTarget">
    <double>29.000000000000000</double>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>808</x>
     <y>269</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 14px;
color: white;</string>
   </property>
   <property name="text">
    <string>90</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>564</x>
     <y>205</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 14px;
color: white;</string>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>808</x>
     <y>142</y>
     <width>21</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: bold 14px;
color: white;</string>
   </property>
   <property name="text">
    <string>70</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>587</x>
     <y>97</y>
     <width>231</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image:url(&quot;:/led/ledOFF.png&quot;);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>70</y>
     <width>304</width>
     <height>295</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image:url(&quot;:/other/ring.png&quot;);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>201</x>
     <y>390</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black;
background-color:rgb(131, 131, 131);
color: white;
font: bold 20px;
qproperty-alignment: AlignCenter;</string>
   </property>
   <property name="text">
    <string>LEVEL 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>383</x>
     <y>390</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black;
background-color:rgb(131, 131, 131);
color: white;
font: bold 20px;
qproperty-alignment: AlignCenter;</string>
   </property>
   <property name="text">
    <string>LEVEL 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>390</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black;
background-color:rgb(131, 131, 131);
color: white;
font: bold 20px;
qproperty-alignment: AlignCenter;</string>
   </property>
   <property name="text">
    <string>TEMPERATURE</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>backButton</zorder>
  <zorder>levelProggresBar1</zorder>
  <zorder>levelProggresBar2</zorder>
  <zorder>label_10</zorder>
  <zorder>label_8</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>label_2</zorder>
  <zorder>label_5</zorder>
  <zorder>label_12</zorder>
  <zorder>temperatureValue</zorder>
  <zorder>label_3</zorder>
  <zorder>temperatureBar</zorder>
  <zorder>label_11</zorder>
  <zorder>label_4</zorder>
  <zorder>label_9</zorder>
  <zorder>label_13</zorder>
  <zorder>label_14</zorder>
  <zorder>label_15</zorder>
  <zorder>label_16</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
